docs/eventing/getting-started.md

```diff
@@ -213,37 +138,28 @@ specific consumer.
    END
    ```
 
-1. Just like you did with the `Broker`, verify that your event consumers are
-   working by running the following command:
-
-   ```sh
+1. Verify that the event consumers are working by entering the following command:
+   ```
    kubectl --namespace event-example get deployments hello-display goodbye-display
    ```
-
    This lists the `hello-display` and `goodbye-display` consumers that you
    deployed:
-
-   ```sh
+   ```
    NAME              READY   UP-TO-DATE   AVAILABLE   AGE
    hello-display     1/1     1            1           26s
    goodbye-display   1/1     1            1           16s
    ```
+   The number of replicas in the **READY** column should match the number of replicas in the **AVAILABLE** column, which might take a few minutes.
```

Comment:
Yikes, again here, nothing should be taking a few minutes :) Maybe like few seconds.

Reply:
Thanks :)